src/Http/Middleware/Telemetry.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Enjin\Platform\Http\Middleware;
+
+use Closure;
+use Enjin\Platform\Enums\Global\PlatformCache;
+use Enjin\Platform\Services\PhoneHomeService;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Cache;
+use Illuminate\Support\Facades\Log;
+use Symfony\Component\HttpFoundation\Response;
+use Throwable;
+
+class Telemetry
+{
+    public function __construct(protected PhoneHomeService $service) {}
+
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Closure(\Illuminate\Http\Request): (\Symfony\Component\HttpFoundation\Response)  $next
+     */
+    public function handle(Request $request, Closure $next): Response
+    {
+        if (!config('telemetry.enabled') || config('telemetry.enabled') === 'false') {
```

Comment:
Refer to `isEnabled()` function addition mentioned on `PhoneHomeService`. From there, we can probably wrap the `try/catch` in an `if` statement so we don't repeat `return $next($request);`. It is good that this check if here, though, given that we do not watch to entertain calling `Cache` for this lock value if it's disabled.